bump to 20180501

carlodri opened the pull request.

carlodri committed:
bump to 20180427

carlodri commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.1.2 and pinning 2018.04.26

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

carlodri committed:
add nasm and remove yasm

jakirkham commented on recipe/meta.yaml:
```diff
@@ -1,23 +1,23 @@
   fn: x264-snapshot-{{ version }}-2245-stable.tar.bz2
   url: http://download.videolan.org/pub/videolan/x264/snapshots/x264-snapshot-{{ version }}-2245-stable.tar.bz2
-  md5: cfd27222c7aa9983259ddb10f7f570a6
+  md5: 89fb34fe9d0959894592e308cafff5ad
```
Could we switch to a more secure checksum?

jakirkham commented on recipe/meta.yaml:
```diff
@@ -1,23 +1,23 @@
 requirements:
   build:
     - toolchain
     - perl 5.20.3.1
```
With the new `conda-smithy`, we can drop this hacky `perl` pinning.

jakirkham commented on recipe/meta.yaml:
```diff
@@ -1,23 +1,23 @@
     - toolchain
     - perl 5.20.3.1
-    - yasm
+    - nasm
```
Note to others: Upstream switched from `yasm` to `nasm`. ( https://stackoverflow.com/a/44573541 )

jakirkham commented on recipe/meta.yaml:
```diff
@@ -1,23 +1,23 @@
 
 requirements:
   build:
     - toolchain
```
Also we can change this to `{{ compiler("c") }}` and then re-render. This is part of a compiler transition that we are making throughout conda-forge.

jakirkham commented:
Thanks @carlodri.

Could you please resolve merge conflicts with `master`? Made a few other comments above. Hopefully nothing too complicated.

Any interest in helping out with maintenance of this feedstock?

carlodri committed:
use sha256 and remove perl version pin

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict.
Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the @ notation in a comment) if you believe this is a bug.

carlodri committed:
Merge branch 'master' into patch-1

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

carlodri committed:
add myself as maintainer [ci skip]

carlodri committed:
add myself as maintainer

carlodri committed:
Merge branch 'patch-1' of https://github.com/carlodri/x264-feedstock into patch-1

carlodri commented:
@jakirkham thanks for the feedback! I've hopefully fixed merge conflicts and added myself as maintainer

carlodri commented:
Oh sorry, still missed the compiler thing, will do tomorrow...

jakirkham committed:
Use the conda-build 3 compiler syntax

Switches from using the `toolchain` to the `conda-build` 3 `compiler`
syntax. This still uses `toolchain` under the hood while allowing us to
move to a syntax that is more inline with `AnacondaRecipes`. Also
provides the compiler run time as a dependency on Linux and Windows
(macOS still pending on this change).

jakirkham commented:
@conda-forge-admin, please re-render.

jakirkham commented:
Thanks @carlodri. Looks good.

Went ahead and added the `compiler` change to your PR. Bot to re-render. Hope that is ok.

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.1.2 and pinning 2018.04.28

jakirkham commented:
Would like to sort out how we pin `x264` in `ffmpeg` before merging this. Have raised issue ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/51 ) to discuss how best to pin `x264`.

carlodri commented:
@jakirkham can this be merged? I think `ffmpeg` shouldn't have problems with the new version, but we can try and if things go wrong we pin accordingly.

jakirkham commented:
So I'm 99% certain this will break `ffmpeg` as the SONAME has changed a dozen times (amongst other breaking changes) since the release in here and the current one. I'd rather us start by determining how we want to pin things ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/51 ) and apply that pinning where needed before updating. It may still break things unfortunately, but we will be down to pulling unpinned `ffmpeg` packages at that point, which is something we will need to do anyways.

ref: https://abi-laboratory.pro/tracker/timeline/x264/

carlodri committed:
bump version

carlodri committed:
move nasm to host

carlodri committed:
fix sha256

carlodri commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.1.6 and pinning 2018.06.29

carlodri commented:
supersedes and includes changes from #15
